native/src/components/CategoryListItem.tsx

```diff
@@ -8,10 +8,9 @@ import dimensions from '../constants/dimensions'
 import { LanguageResourceCacheStateType } from '../utils/DataContainer'
 import { getCachedThumbnail } from './Categories'
 import SimpleImage from './SimpleImage'
-import StyledLink from './StyledLink'
 import SubCategoryListItem from './SubCategoryListItem'
 
-const FlexStyledLink = styled(StyledLink)`
+const FlexStyledLink = styled.TouchableOpacity`
   display: flex;
   flex-direction: column;
```

Comment:
```suggestion
  flex-direction: column;
  margin: 0 20px;
```

Reply:
Without the margin, the links stretch the entire screen.
<img width="332" alt="image" src="https://github.com/digitalfabrik/integreat-app/assets/18513943/a0711612-c8d4-4ce4-97c4-174646a19d9e">

Reply:
I can't reproduce this, for me it looks like this:
< src="https://github.com/digitalfabrik/integreat-app/assets/29095487/6539025b-488d-4ba5-a762-8ae1021af435" width="400px" />